src/map/map.ts

```diff
@@ -925,7 +739,10 @@ export class PropertiesMap {
         }
         
         // make a copy of the default layout
-        const layout = {...DEFAULT_LAYOUT};
+        //const layout = {...DEFAULT_LAYOUT};
+        // makes a deep copy - otherwise DEFAULT_LAYOUT entries get overwritten 
+        const layout = JSON.parse(JSON.stringify(DEFAULT_LAYOUT)); 
```

Comment:
Good catch! The previous code only did a copy of top level items =/. You can also remove the old code here!